compiler/src/bin/native/dune

```diff
@@ -4,12 +4,6 @@
  (implements serve)
  (preprocess
   (pps ppx_blob))
- (preprocessor_deps client/client.bc.js)
- (libraries
-  slipshow
-  fpath
-  lwt
-  inotify.lwt
-  dream
-  ; bos
-  ))
+ (preprocessor_deps
+  (file client/client.bc.js))
```

Comment:
From the dune manual, `(file <filename>)` and `<filename>` is the same. What is the reason to prefer on to the other?

Reply:
Oh that was from testing the ppx_blob issue I mentioned above